social-tracking-protection-blacklist.json

```diff
@@ -0,0 +1,73 @@
+{
+ "license": "https://www.mozilla.org/MPL/",
+ "categories": {
+     "Facebook": [
+         {
+             "facebook.com": {
+                 "facebook.com": [
```

Comment:
Should we add whatsapp domains?

Reply:
I see that whatsapp.com is already on the bottom of the list. Are you suggesting Luke add other domains from Whatsapp?